recipe/meta.yaml

```diff
@@ -1,99 +1,238 @@
+  # url: https://poppler.freedesktop.org/poppler-{{ version }}.tar.xz
+  sha256: d9aa9cacdfbd0f8e98fc2b3bb008e645597ed480685757c3e7bc74b4278d15c0
+  patches:                            # [win]
+    - exportsymbols.patch             # [win]
+    - windows-data.patch              # [win]
+    # libtiff uses Unix I/O even on Windows
+    # https://github.com/conda-forge/libtiff-feedstock/pull/51
+    - disable-libtiff-win32-io.patch  # [win]
+    - includesystembeforejpeg.patch   # [win]
 
 build:
-  skip: True  # [win]
   number: 0
-  run_exports:
-    # add new symbols often.  Don't remove symbols often at all.  Keeping default.
-    #   https://abi-laboratory.pro/tracker/timeline/poppler/
-    - {{ pin_subpackage('poppler') }}
   detect_binary_files_with_prefix: true
-  skip: true  # [s390x or (unix and (cxx_compiler == 'toolchain_cxx'))]
   missing_dso_whitelist:
    - $RPATH/libm.so.6
    - $RPATH/libc.so.6
    - $RPATH/ld-linux-x86-64.so.2
    - $RPATH/libpthread.so.0
-  ignore_run_exports:
-   - libxcb
+   - $RPATH/ld64.so.1
 
 requirements:
   build:
     - {{ compiler('c') }}
     - {{ compiler('cxx') }}
-    - pkg-config  # [not win]
-    - cmake
-    - ninja
-    - {{ posix }}patch
-    - {{ posix }}make
-    - perl
+    - binutils  # [linux]
+    # Need these CDTs for Qt on Linux. Please keep them alphabetized!
+    - {{ cdt('libselinux-devel') }}      # [linux]
+    - {{ cdt('libxdamage') }}            # [linux]
+    - {{ cdt('libxfixes') }}             # [linux]
+    - {{ cdt('libxscrnsaver-devel') }}   # [linux and not ppc64le]
+    - {{ cdt('libxtst-devel') }}         # [linux and not ppc64le]
+    - {{ cdt('libxxf86vm') }}            # [linux]
+    - {{ cdt('mesa-libgl-devel') }}      # [linux and not s390x]
+    - {{ cdt('mesa-libegl-devel') }}     # [linux and not ppc64le]
+    - {{ cdt('mesa-libgbm') }}           # [linux and not ppc64le]
+    - {{ cdt('mesa-dri-drivers') }}      # [linux]
     - {{ cdt('libx11-devel') }}          # [linux]
     - {{ cdt('libxau-devel') }}          # [linux]
     - {{ cdt('libxext-devel') }}         # [linux]
     - {{ cdt('libxrender-devel') }}      # [linux]
     - {{ cdt('xorg-x11-proto-devel') }}  # [linux]
+    - m2-msys2-runtime  # [win]
+    - pkg-config
+    - cmake
+    - {{ posix }}make
+    - ninja
+    - {{ posix }}patch
+    - perl 5.*
+    - gobject-introspection 1.*  # [not win]
   host:
-    - python >=3.6
-    - boost-cpp
-    - cairo  # [not win]
+    - python 3.9
+    - boost-cpp {{ boost_cpp }}
+    - cairo
     - curl
-    - freetype
-    - glib
-    - gobject-introspection 1.*  # [not win]
-    - jpeg
-    - libiconv                           # [osx]
-    - libpng
-    - libtiff
-    - libxcb                             # [linux]
-    - openjpeg >=2
-    - zlib
     - fontconfig
-    - libcurl
-    - gettext  # [osx]
-  run:
-    - poppler-data
-    - cairo  # [not win]
-    - curl
     - freetype
+    - gettext  # [osx]
```

Comment:
Seems like gtk and gdk-pixbuf are required dependencies, see https://gitlab.freedesktop.org/poppler/poppler/-/blob/master/CMakeLists.txt#L142 and https://gitlab.freedesktop.org/poppler/poppler/-/blob/master/cmake/modules/FindGTK.cmake#L19.
Do we need them?

Reply:
I assume no. We use either X11 qt instead.  we might consider support for gtk3, but I would consider this as future plan. nothing to tackle now